drgn_tools/kernfs_memcg.py

```diff
@@ -198,17 +213,28 @@ def dump_page_cache_pages_pinning_cgroups(
                 if max_pages and found_count == max_pages:
                     break
         except FaultError:
+            fault_count = fault_count + 1
             continue
 
     print(
-        f"Scanned {total_count} pages, found {found_count} pages with memory cgroup refs."
+        f"Scanned {total_count} pages, found {found_count} pages with memory cgroup refs, found {fault_count} faults."
     )
 
 
+class NumMemCgroups(CorelensModule):
+    """Print number of active and dying memcgroups"""
+
+    name = "num-memcgroups"
+
+    def run(self, prog: Program, args: argparse.Namespace) -> None:
+        get_num_mem_cgroups(prog)
+
+
 class PagesPinningMemcgroups(CorelensModule):
     """Print information related to pages, that are pinning memcgroup(s)"""
 
-    name = "kernfs_memcg"
+    name = "pages-pinning-memcg"
+    run_when = "never"
```

Comment:
It makes senes this only run when user requests it explicitly. Should we make the default behavior to dump all pages pinning zombie cgroup, that looks more common to me when troubleshooting zombie cgroup issues?

Reply:
Makes sense. I have modified it to dump all pages by default.